AccountingSystem/Service/ReportGeneratorHandler.py

```diff
@@ -28,13 +29,13 @@ class ReportGeneratorHandler(RequestHandler):
   """
 
   __acDB = None
-  __reportRequestDict = {'typeName': types.StringTypes,
-                         'reportName': types.StringTypes,
+  __reportRequestDict = {'typeName': six.string_types,
```

Comment:
FYI: I reverted these changes in #4240 ad this has the potential to break.

For example `type(obj) in valid_types` only works with the first one of:
```python
valid_types = types.StringTypes
# valid_types == (str, unicode)
valid_types = six.string_types
# valid_types == (basestring,)
```

There might also be other ways this issue can manifest. That said I think it's okay for v7r2 as I've fixed all of these in DIRAC but I haven't looked elsewhere.